### PR TITLE
adding react native example, and reorganizing sdk directories

### PR DESCRIPTION
**Pull request by daxfrost**
* Adding React Native example
* Reorganizing SDK directories

See README: https://github.com/TextureHQ/examples/pull/12/files#diff-75d3a45cf11e67eec54e5bc03b88d5275a447e11cba4c0ea67e62c028b0e07c3

### CODE REVIEW

File: connect/sdk/javascript/.env.example
Comment (rcasto) [inline connect/sdk/javascript/.env.example:1]: Might wanna update the folder name here to `javascript-example` to be consistent with the others

**Review by victorquinn — APPROVED** [review]
🐰

File: connect/sdk/react-native-example/android/app/src/main/res/mipmap-xxxhdpi/ic_launcher_round.png
Comment (rcasto) [inline connect/sdk/react-native-example/android/app/src/main/res/mipmap-xxxhdpi/ic_launcher_round.png:1]: Know a lot of these are generated or come boiller plate with the expo app start, not sure if we'd want to use some Texture logos and theme colors here at some point to pretty these up a bit haha

File: connect/sdk/react-native-example/package-lock.json
Comment (rcasto) [inline connect/sdk/react-native-example/package-lock.json:1]: Not a biggie, but for the other examples we are using yarn, might wanna consolidate these to one or the other

**Review by rcasto — APPROVED** [review]
Just a few minor comments, but looks good! :shipit:

File: connect/sdk/react-native-example/android/app/src/main/res/mipmap-xxxhdpi/ic_launcher_round.png
Comment (daxfrost) [reply connect/sdk/react-native-example/android/app/src/main/res/mipmap-xxxhdpi/ic_launcher_round.png:1]: hehe I thought the same, but on the other hand I feel it's more clear to developers this is a vanilla project if we don't.

**Pull request merged**